cmake/toolchain-mingw-i686.cmake

```diff
@@ -14,8 +14,8 @@ set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 
 # Prefer posix gcc variant for gtest pthread support
 find_program(C_COMPILER_FULL_PATH NAMES
-    ${CMAKE_C_COMPILER_TARGET}-w64-mingw32-gcc-posix
-    ${CMAKE_C_COMPILER_TARGET}-w64-mingw32-gcc
+    ${CMAKE_C_COMPILER_TARGET}-gcc-posix
+    ${CMAKE_C_COMPILER_TARGET}-gcc
     gcc-mingw32-w64-${CMAKE_C_COMPILER_TARGET})
```

Comment:
This like needs also be corrected...

Reply:
I'm not aware of any distro that uses such a naming scheme, so I'll remove it.